src/main/java/wolox/training/models/Book.java

```diff
@@ -28,7 +27,7 @@
     /**
      * Author of the book
      */
-    @Column(nullable = false)
+    @Column(nullable = false, unique = true)
```

Comment:
Just curious as to why the author should be unique? 💭

Reply:
cause like y put Optional<Book> findByAuthor to accomplish with the request in the card that told return just one book in the method findByAuthor, if the bd have more than one book with the same author that give me an error when i try to get a book by author 😬